### PR TITLE
Enable grid for premium pages

### PR DESCRIPTION
**Pull request by aeris**

### CODE REVIEW

File: docs/buttons.html [hunk 1]
```diff
@@ -26,7 +26,7 @@ <h2 class="h4">Primary</h2>
         </div>
         <div class="my-3">
           <h2 class="h4">Secondary</h2>
-          <button type="button" class="btn btn-outline-info">Button</button>
+          <button type="button" class="btn btn-secondary">Button</button>
```
Comment (nono) [inline docs/buttons.html:29]: Please keep the `btn-outline-info` here. It makes more sense with bootstrap (the secondary title is a reference to Material-UI and Joël's documentation).

File: postcss-cozy-vars/index.js [hunk 2]
```diff
@@ -99,8 +99,8 @@ function changeColor(decl) {
     case ".btn:hover":
     case ".btn-primary":
     case ".btn-primary:hover":
-    case ".btn-secondary":
-    case ".btn-secondary:hover":
+    // case ".btn-secondary":
+    // case ".btn-secondary:hover":
```
Comment (nono) [inline postcss-cozy-vars/index.js:103]: If you want to remove code, remove it, don't comment it

File: src/cozy/_colors.scss [hunk 3]
```diff
@@ -24,13 +24,13 @@ html {
   --primaryColorLightest: #eaf3ff;             // Primary 100
   --primaryTextColor: rgba(29, 33, 42, .9);    // Grey 900 + alpha 0.9
   --primaryTextContrastColor: #fff;            // Primary contrastText
-  --secondaryColor: #a346cd;                   // Secondary 600
-  --secondaryColorDark: #7a349a;               // Secondary 700
+  --secondaryColor: transparent;                   // Secondary 600
+  --secondaryColorDark: transparent;               // Secondary 700
```
Comment (nono) [inline src/cozy/_colors.scss:28]: I really don't think that the comment is still acurate after this change

**Review by nono — CHANGES_REQUESTED** [review]
This PR cannot be accepted currently:

- the tests are broken
- there are 2 changes (1. secondary button & 2. grid), so we should have 2 commits
- the commit message must follow the semantic-release rules

**Review by nono — CHANGES_REQUESTED** [review]
I don't understand what you want to do. Why make the `btn-secondary`, `btn-outline-secondary`, and `btn-outline-info` look the same?

And, it is not OK to change the color of `text-secondary`. It is a color chosen by our designer.

**Review by nono — COMMENTED** [review]
Can you explain why it would be helpful to have the grid module? Can't we just use a few CSS grid rules? I think bootstrap grid module is quite big, and I would prefer to avoid the bloat if we just need a very small part of it.

And the commit message must follow the semantic-release rules (bis).

**Comment by aeris** [comment]
After investigation, we will use directly bootstrap-grid from npm package for premium.
So this PR useless now.

**Pull request closed without merging**